AssetManagementTestProject/TestCases/US302_LoginTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.Services;
+using AssetManagementTestProject.TestData;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+
+namespace AssetManagementTestProject;
+
+[TestFixture]
+public class US302_LoginTest : NUnitWebTestSetup
+{    
+    protected ChangePassword1stTimePage? ChangePw1stTime;
+
+
+    [TestCase(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN)]
+    [TestCase(Constant.STAFF_USERNAME, Constant.STAFF_PASSWORD)]
+    public void TC01_UserLoginSuccess(string username, string password)
+    {
+        LoginPage?.Login(username, password);
+        DriverBaseAction?.WaitToBeVisible(HomePage.HeaderHomePage);
+        Asserter?.AssertElementIsDisplayed(HomePage.HeaderHomePage);
+    }
+    [Test]
+    public void TC02_UserAskedChangePasswordFirstTime()
+    {
+        AuthorizationService = new AssetManagementAPIServices();
+        Token = AuthorizationService.ReturnLoginToken(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN);
+        APIService = new AssetManagementAPIServices();
+        NewUser = APIService.ReturnNewUser(Constant.NEW_ADMIN_HN, Token);
+        FirstTimeLoginData newLoginData = new FirstTimeLoginData();
+        newLoginData.NewUser = NewUser;
+        string newAdminUsername = newLoginData.GetUsername();
+        string newAdminPassword = newLoginData.GetPassword();
```

Comment:
This is common action => PrepareNewUser()
Can we move it to the parent test class?

Reply:
Done